run-basic-batch-operators-test.sh

```diff
@@ -232,6 +232,44 @@ wait_package_ok() {
 	done
 }
 
+wait_all_packages_ok() {
+	local \
+		start_time \
+		timeout_seconds=600
+
+	prev_count=$(get_packeges)
```

Comment:
Should all variables be local?

Reply:
Could be. I followed other function structure, so I didn't create all of them a local, but it could be changed.
What do you think?

Reply:
Yes, a new function should be of the best quality. We will fix other functions in a separate PR.